pkg/fileutil/fs_case.go

```diff
@@ -24,6 +24,13 @@ func IsCaseInsensitiveLocation(fs FS, dirPath string, warnFunc func(string)) (bo
 	if err != nil {
 		return false, fmt.Errorf("generate random name: %w", err)
 	}
+	exists, err := fs.Exists(dirPath)
```

Comment:
Not part of this pr, I think?

Reply:
It's in the PR description (😄). It just bothered me so much and the two changes are small enough to allow myself to push it into the same PR.